Inheritance/src/character/extendsCharacter/extendsPawPatrol/Chase.java

```diff
@@ -0,0 +1,21 @@
+package character.extendsCharacter.extendsPawPatrol;
+
+import character.extendsCharacter.PawPatrol;
+
+public class Chase extends PawPatrol {
+    public Chase(String name, String function, String superPower, String breed) {
+        super(name, function, superPower, breed);
+    }
+    public Chase() {
+        super("character.extendsCharacter.extendsPawPatrol.Chase",
```

Comment:
character.extendsCharacter.extendsPawPatrol.Chase - this should be only Chase

Reply:
Done